docs/installation_guide.md

```diff
@@ -1,163 +1,238 @@
+# Installation Guide
+
+TODO:
+  - Add improved / automated way to fetch rabbitmq password 
+  - Add description for PVC part
+  - 
+
 This document provides guidelines to installing all dependencies of DYNAMOS.
 
-**NOTE**: This document mainly focuses on Linux based commands. Thus we expect Windows users to enable Windows Subsystem for Linux 2. If there are any special instructions related to other operating systems, it will clearly be highlighted.
+**NOTE**: This document mainly focuses on Linux Debian based commands. Thus we expect Windows users to enable Windows Subsystem for Linux 2. If there are any special instructions related to other operating systems, it will clearly be highlighted.
+
+# Prerequisite software tools
+In order to setup DYNAMOS, we need to install a few tools as a foundation. This section covers such tools.
+
+## WSL2 (for Windows)
+Since we prefer to use Linux based operating systems, we suggest using the Windows Subsystem for Linux on a Windows machine. This will decrease the possibility of diverging issues when setting up the environment.
```

Comment:
This is not really a preference though, but a requirement. I don't believe you can run unix based containers on Windows without WSL. So the wording can be a bit harsher here.

Reply:
Makes sense, will update the wording